src/electron/main.js

```diff
@@ -189,9 +195,17 @@ app.whenReady().then(async () => {
 
   ipc.on('gitify:icon-idle', () => {
     if (!mb.tray.isDestroyed()) {
-      mb.tray.setImage(
-        updateAvailableMenuItem.visible ? idleUpdateAvailableIcon : idleIcon,
-      );
+      if (process.platform === 'darwin' || process.platform === 'win32') {
+        mb.tray.setImage(
+          updateAvailableMenuItem.visible ? idleUpdateAvailableIcon : idleIcon,
+        );
+      } else {
```

Comment:
Not 100% certain that all linux distros should use the white icon set 🤷‍♂️

Reply:
This _could_ be a user controlled setting, which would be more versatile, but adds some complexity to the impl